package.json

```diff
@@ -65,5 +65,12 @@
     "prettier": "^2.8.8",
     "turbo": "^1.9.6",
     "typescript": "^5.0.4"
+  },
+  "resolutions": {
+    "@typescript-eslint/eslint-plugin": "6.0.0",
```

Comment:
Did we need these now with the stable relewse?

Reply:
maybe not! https://github.com/t3-oss/create-t3-app/pull/1476#discussion_r1260288775